machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc

```diff
@@ -19,7 +19,7 @@ You can add more compute machines to your {product-title} cluster on bare metal.
 If you do not have access to the {op-system-first} images that were used to create your cluster, you can add more compute machines to your {product-title} cluster with newer versions of {op-system-first} images. For instructions, see link:https://access.redhat.com/solutions/5514051[Adding new nodes to UPI cluster fails after upgrading to OpenShift 4.6+].
 ====
 
-[id="creating-machines-bare-metal"]
+[id="creating-rhcos-machines-bare-metal"]
```

Comment:
Adding `rhcos` to this anchor ID to distinguish it from the ID and filename used in the added module (`modules/creating-machines-bare-metal.adoc`)